src/praisonai-agents/tests/unit/agent/test_scientific_writer_agent.py

```
@@ -0,0 +1,323 @@
+            from praisonaiagents import ScientificWriterAgent
+            
+            agent = ScientificWriterAgent(
+                name="Test Scientific Writer",
+                model="gpt-4o-mini",  # Use a standard model for testing
+                instructions="You are a test scientific writer."
+            )
+            
+            # Verify agent properties
+            assert agent.agent.name == "Test Scientific Writer"
+            assert "scientific writer" in agent.agent.instructions.lower()
+            assert agent.is_cajal_model == False  # gpt-4o-mini is not CAJAL
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `False`; use `not agent.is_cajal_model:` for false checks

Replace with `not agent.is_cajal_model`

(E712)